march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright 2020 Project March.
+#include <hardware_interface/joint_command_interface.h>
+#include <hardware_interface/joint_state_interface.h>
+#include "march_joint_inertia_controller/inertia_estimator.h"
+#include "math.h"
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+void absolute(std::vector<double> a, std::vector<double>& b)
+{
+  for (size_t i = 0; i < a.size(); ++i)
+  {
+    b[i] = abs(a[i]);
+  }
+}
+
+double mean(std::vector<double> a)
+{
+  double sum = 0.0;
+  for (size_t i = 0; i < a.size(); ++i)
+  {
+    sum += a[i];
+  }
+  if (sum == 0.0)
+  {
+    return 0.0;
+  }
```

Comment:
This check does not add anything, right? It might be a bit faster in an edge case, but is it worth it? If you do this, because you do not want to divide by zero, you should check whether the size is zero.

Reply:
you are correct, I will change this